src/Task3.kt

```diff
@@ -0,0 +1,16 @@
+
+//Made it function in order to manipulate the array
+fun hasDuplicates(array : IntArray): Boolean {
+    if(array.count() < 2) return false
+
+    array.forEach{
+        val indexNormalized = Math.abs(it) -1
```

Comment:
Why do you need to abs it ? Problem states:
`Given an array of numbers having size N in which every number is between 1 and N`

Reply:
The implementation I've used makes them negative to check them as seen